app/models/manageiq/providers/ansible_tower/shared/automation_manager/amazon_credential.rb

```diff
@@ -23,4 +23,10 @@ module ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::AmazonCrede
   }.freeze
 
   API_ATTRIBUTES = COMMON_ATTRIBUTES.merge(EXTRA_ATTRIBUTES).freeze
+
+  API_OPTIONS = {
+    'display_name' => 'Amazon',
```

Comment:
We use `label` for the individual fields in the credential as [here](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/ansible_tower/shared/automation_manager/amazon_credential.rb#L11). It will be of same usage as `display_name`. Do we want to unify them?